Makefile

```diff
@@ -4,7 +4,7 @@
 cppdock:
 	docker build --force-rm=true -t ricejasonf/cppdock .
 
-platforms: platform_linux_x64 platform_emscripten platform_tvossimulator
+platforms: platform_linux_x64 platform_emscripten platform_tvossimulator platform_macosx
```

Comment:
Please remove both osx and tvossim for the `platforms` target, and I will just merge this into `develop`.